pkg/component/watch.go

```diff
@@ -16,6 +16,24 @@ import (
 	"github.com/pkg/errors"
 )
 
+func isIgnore(path string, patterns []string) bool {
+	for _, pattern := range patterns {
+		if matched, _ := regexp.MatchString(pattern, path); matched {
+			return true
+		}
+	}
+	return false
+}
+
+func stringinSlice(str string, strSlice []string) bool {
```

Comment:
need tests

Reply:
Removing it in updated PR no more in use